docs/readmes/orc8r/dev_minikube.md

```diff
@@ -121,7 +122,7 @@ helm dep update
 helm upgrade --install --namespace orc8r --values ${MAGMA_ROOT}/orc8r/cloud/helm/lte.values.yaml lte .
 ```
 
-If successful, you should get something like this
+If successful, you should get something like this. (It may take a couple minutes before all pods are finished being created)
```

Comment:
```suggestion
If successful, you should get something like this (it may take a couple minutes before all pods are finished being created)
```

Reply:
I moved the phrase to the beginning of the sentence so that it doesn't block the this! I'm down to change if you think it's better some other way though :)